common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -460,6 +460,7 @@ def generate_report_data(self, user_state_iterator, limit_responses=None):
                     'seed': user_state.state.get('seed'),
                 },
                 seed=user_state.state.get('seed'),
+                minimal_init=True,
```

Comment:
LGTM. Just wondering are there any particular XBlock cases for which test(s) should be added here?:
https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/tests/test_capa_module.py#L3139-L3138

Reply:
Yup! I was looking at that and trying to figure out what a meaningful test would be, since there's so much mocking of CapaProblem internals in it. I think I'm going to settle on making a test for "invoking this doesn't cause codejail to start", and I'm seeing if adding a script tag to the data attrib is enough to start that.